Problem2/largest_two_numbers.rb

```diff
@@ -0,0 +1,29 @@
+# File: largest_two_numbers.rb
+
+class LargestTwoNumbers
```

Comment:
I see you can solve this. How about having some fun with ruby and trying to make it as short as possible?